src/detect-engine-event.c

```diff
@@ -68,17 +134,23 @@ void DetectEngineEventRegister (void)
 #ifdef UNITTESTS
     sigmatch_table[DETECT_ENGINE_EVENT].RegisterTests = EngineEventRegisterTests;
 #endif
+    sigmatch_table[DETECT_ENGINE_EVENT].SupportsPrefilter = PrefilterEngineEventIsPrefilterable;
```

Comment:
engine-event is only used by detect events, so mostly (or only?) swf decoding. This can run after the prefilter engines run. So events may be missed.

Reply:
ah yes, I forgot https://redmine.openinfosecfoundation.org/issues/4482 cc @jlucovsky

Reply:
So, I keep the PR as is, right ?

Reply:
No, the engine event should not use this in any case, as the prefilter runs before the events might be generated I think

Reply:
There are events generated before the prefilter runs like `engine-event:stream.3whs_synack_tfo_data_ignored`

I take them out anyways...